The Docs workflow still fails:
```
    Unexpected failing examples (1):
    
        ../examples/tutorials/advanced/draping_on_3d_surface.py failed leaving traceback:
    
        Traceback (most recent call last):
          File "/home/runner/work/pygmt/pygmt/examples/tutorials/advanced/draping_on_3d_surface.py", line 103, in <module>
            with rasterio.open(url_to_image) as dataset:
                 ~~~~~~~~~~~~~^^^^^^^^^^^^^^
          File "/home/runner/micromamba/envs/pygmt/lib/python3.14/site-packages/rasterio/env.py", line 464, in wrapper
            return f(*args, **kwds)
          File "/home/runner/micromamba/envs/pygmt/lib/python3.14/site-packages/rasterio/__init__.py", line 367, in open
            dataset = DatasetReader(path, driver=driver, sharing=sharing, thread_safe=thread_safe, **kwargs)
          File "rasterio/_base.pyx", line 329, in rasterio._base.DatasetBase.__init__
        rasterio.errors.RasterioIOError: CURL error: Failed to connect to oceania.generic-mapping-tools.org port 443 after 133284 ms: Could not connect to server
```
~~it seems it still use the default data server.~~

Actually, it's because this example explicitly uses the oceania URL. Now fixed f09b45023e154f74b9b26a63978cd9cbc15923f2

--- a/.github/workflows/benchmarks.yml
+++ b/.github/workflows/benchmarks.yml
@@ -86,7 +86,7 @@ jobs:
       - name: Run benchmarks
         uses: CodSpeedHQ/action@9d332c4d90b43981c3e55ae8e38e68709996240f # v4.17.0
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         with:
           mode: "instrumentation"
           # 'bash -el -c' is needed to use the custom shell.

--- a/.github/workflows/cache_data.yaml
+++ b/.github/workflows/cache_data.yaml
@@ -71,7 +71,7 @@ jobs:
       # Download remote files
       - name: Download remote data
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: |
           python -c "from pygmt.helpers.caching import cache_data; cache_data()"
 

--- a/.github/workflows/ci_docs.yml
+++ b/.github/workflows/ci_docs.yml
@@ -136,7 +136,7 @@ jobs:
 
       - name: Build the HTML documentation
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: |
           log_file="${RUNNER_TEMP}/sphinx-html.log"
           make -C doc clean html 2>&1 | tee "${log_file}"
@@ -149,7 +149,7 @@ jobs:
 
       - name: Build the PDF documentation
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: |
           log_file="${RUNNER_TEMP}/sphinx-pdf.log"
           make -C doc pdf 2>&1 | tee "${log_file}"

--- a/.github/workflows/ci_doctests.yaml
+++ b/.github/workflows/ci_doctests.yaml
@@ -85,7 +85,7 @@ jobs:
       # Run the doctests
       - name: Run doctests
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: |
           log_file="${RUNNER_TEMP}/pytest.log"
           make doctest PYTEST_EXTRA="-r P" 2>&1 | tee "${log_file}"

--- a/.github/workflows/ci_tests.yaml
+++ b/.github/workflows/ci_tests.yaml
@@ -190,7 +190,7 @@ jobs:
       # Run the regular tests
       - name: Run tests
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: |
           log_file="${RUNNER_TEMP}/pytest.log"
           make test PYTEST_EXTRA="-r P --reruns 2" 2>&1 | tee "${log_file}"

--- a/.github/workflows/ci_tests_dev.yaml
+++ b/.github/workflows/ci_tests_dev.yaml
@@ -184,7 +184,7 @@ jobs:
       - name: Test with pytest
         run: make test PYTEST_EXTRA="-r P --reruns 2"
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
           GMT_LIBRARY_PATH: ${{ runner.temp }}/gmt-install-dir/lib
 
       # Upload diff images on test failure

--- a/.github/workflows/ci_tests_legacy.yaml
+++ b/.github/workflows/ci_tests_legacy.yaml
@@ -96,5 +96,5 @@ jobs:
       # Run the tests but skip images
       - name: Run tests
         env:
-          GMT_DATA_SERVER: https://mirrors.ustc.edu.cn/gmtdata/
+          GMT_DATA_SERVER: https://opentopography.s3.sdsc.edu/gmtdata
         run: make test_no_images PYTEST_EXTRA="-r P"

--- a/examples/tutorials/advanced/draping_on_3d_surface.py
+++ b/examples/tutorials/advanced/draping_on_3d_surface.py
@@ -99,7 +99,9 @@ region_3d = [*region_2d, grd_relief.min().to_numpy(), grd_relief.max().to_numpy(
 # The original image is available on Wikimedia Commons at
 # https://commons.wikimedia.org/wiki/File:Flag_of_Europe.svg
 # but we use a cached version on the GMT data server.
-url_to_image = "https://oceania.generic-mapping-tools.org/cache/euflag.png"
+url_to_image = (
+    "https://github.com/GenericMappingTools/gmtserver-admin/raw/master/cache/euflag.png"
+)
 with rasterio.open(url_to_image) as dataset:
     data = dataset.read()
     drape_grid = xr.DataArray(data, dims=("band", "y", "x"))

